foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigrationSteps.java

```diff
@@ -0,0 +1,76 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+
+import static org.junit.Assert.assertTrue;
+
+public class AiopsReferencePDPPythonMigrationSteps extends AbstractMigrationTest {
+
+    private AbstractAissembleMigration migration;
+    private File validatedFile;
+    private static Boolean validateMigration(File original, File migrated) {
+        Boolean isMigrated = false;
+
+        try {
+            isMigrated =  FileUtils.contentEquals(original, migrated);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return isMigrated;
+    }
+
+    /**
+     * Function to check whether the migration is necessary.
+     * @param file file to check
+     * @return isFileNameSame - Boolean indicating whether the file names are the same.
+     */
+
+    public static Boolean compareFileNames(File file) {
```

Comment:
Something small but I think you should rename this method to something like "isAissembleSecurityProperties"

Reply:
Updated to the recommended.